tests/predictors/test_td.py

```diff
@@ -239,7 +239,7 @@ def test_run_exploration(self, context: Context) -> None:
 
         flexmock(random)
         random.should_receive("randrange").with_args(1, 2).and_return(0).once()
-        random.should_receive("random").with_args().and_return(
+        random.should_receive("random").and_return(
```

Comment:
Looks like flexmock does not correctly detect number of parameters passed in here.